### ISSUE TITLE
Missing script/eclipse-support/

### ISSUE THREAD

**Issue by ghost** [RadiusNetworks/android-ibeacon-service#10]
Compiling the current git HEAD fails because of a missing script/eclipse-support/ directory.

```
[pjb@mercure :0.0 src]$ git clone https://github.com/RadiusNetworks/android-ibeacon-service.git 
Cloning into 'android-ibeacon-service'...
remote: Counting objects: 674, done.        
remote: Compressing objects: 100% (320/320), done.        
remote: Total 674 (delta 301), reused 607 (delta 242)        
Receiving objects: 100% (674/674), 766.78 KiB | 520 KiB/s, done.
Resolving deltas: 100% (301/301), done.
[pjb@mercure :0.0 src]$ cd android-ibeacon-service/
[pjb@mercure :0.0 android-ibeacon-service]$ gradle -v

------------------------------------------------------------
Gradle 1.8
------------------------------------------------------------

Build time:   2013-09-24 07:32:33 UTC
Build number: none
Revision:     7970ec3503b4f5767ee1c1c69f8b4186c4763e3d

Groovy:       1.8.6
Ant:          Apache Ant(TM) version 1.9.2 compiled on July 8 2013
Ivy:          2.2.0
JVM:          1.7.0_45 (Oracle Corporation 24.45-b08)
OS:           Linux 3.2.0-4-amd64 amd64

[pjb@mercure :0.0 android-ibeacon-service]$ gradle 
:clean UP-TO-DATE
:compileLint
:copyDebugLint UP-TO-DATE
:mergeDebugProguardFiles
:packageDebugAidl UP-TO-DATE
:preBuild UP-TO-DATE
:preDebugBuild UP-TO-DATE
:prepareDebugDependencies
:compileDebugAidl
:compileDebugRenderscript
:generateDebugBuildConfig
:mergeDebugAssets
:mergeDebugResources
:processDebugManifest
:processDebugResources
:generateDebugSources
:compileDebugNote: /home/pjb/src/android-ibeacon-service/src/main/java/com/radiusnetworks/ibeacon/service/IBeaconService.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

:processDebugJavaRes UP-TO-DATE
:packageDebugJar
:packageDebugLocalJar UP-TO-DATE
:packageDebugRenderscript UP-TO-DATE
:packageDebugResources
:bundleDebug
:assembleDebug
:copyReleaseLint UP-TO-DATE
:mergeReleaseProguardFiles
:packageReleaseAidl UP-TO-DATE
:preReleaseBuild UP-TO-DATE
:prepareReleaseDependencies
:compileReleaseAidl
:compileReleaseRenderscript
:generateReleaseBuildConfig
:mergeReleaseAssets
:mergeReleaseResources
:processReleaseManifest
:processReleaseResources
:generateReleaseSources
:compileReleaseNote: /home/pjb/src/android-ibeacon-service/src/main/java/com/radiusnetworks/ibeacon/service/IBeaconService.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

:processReleaseJavaRes UP-TO-DATE
:packageReleaseJar
:packageReleaseLocalJar UP-TO-DATE
:packageReleaseRenderscript UP-TO-DATE
:packageReleaseResources
:bundleRelease
:assembleRelease
:assemble
:check UP-TO-DATE
:build
:bundleEclipse
Generating eclipse bundle from build/libs/AndroidIBeaconLibrary-0.7.1-1-g7e04373.aar to make build/libs/AndroidIBeaconLibrary-0.7.1-1-g7e04373.tar.gz
cp: cannot stat `scripts/eclipse-support': No such file or directory
tar: This does not look like a tar archive
tar: Skipping to next header
tar: Exiting with failure status due to previous errors
mkdir: cannot create directory `build/libs/AndroidIBeaconLibrary/src': No such file or directory
mkdir: cannot create directory `build/libs/AndroidIBeaconLibrary/libs': No such file or directory
mkdir: cannot create directory `build/libs/AndroidIBeaconLibrary/gen': No such file or directory
mv: cannot stat `build/libs/AndroidIBeaconLibrary/*.jar': No such file or directory
tar: AndroidIBeaconLibrary: Cannot stat: No such file or directory
tar: Exiting with failure status due to previous errors
:bundleEclipse FAILED

FAILURE: Build failed with an exception.

* Where:
Build file '/home/pjb/src/android-ibeacon-service/build.gradle' line: 65

* What went wrong:
Execution failed for task ':bundleEclipse'.
> Process 'command 'scripts/bundle-eclipse'' finished with non-zero exit value 2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 10.721 secs

[pjb@mercure :0.0 android-ibeacon-service]$ ls -l build/libs/
total 108
drwxr-xr-x  2 pjb pjb  4096 Dec 13 16:00 ./
drwxr-xr-x 11 pjb pjb  4096 Dec 13 16:00 ../
-rw-r--r--  1 pjb pjb 44427 Dec 13 16:00 AndroidIBeaconLibrary-0.7.1-1-g7e04373-debug.aar
-rw-r--r--  1 pjb pjb 44411 Dec 13 16:00 AndroidIBeaconLibrary-0.7.1-1-g7e04373.aar
-rw-r--r--  1 pjb pjb    45 Dec 13 16:00 AndroidIBeaconLibrary-0.7.1-1-g7e04373.tar.gz
-rw-r--r--  1 pjb pjb   821 Dec 13 16:00 AndroidIBeaconLibrary-debug.ap_
-rw-r--r--  1 pjb pjb   805 Dec 13 16:00 AndroidIBeaconLibrary-release.ap_
[pjb@mercure :0.0 android-ibeacon-service]$ 
```

**Comment by davidgyoung** [comment]
I have confirmed this issue.  It is caused by the files in the directory matching the .gitignore file

**Comment by davidgyoung** [comment]
fixed in c8e25bae81ca019af9ecdd33ffb51e410e55d616
